packages/flutter/example/linux/flutter/generated_plugin_registrant.h

```diff
@@ -2,6 +2,8 @@
 //  Generated file. Do not edit.
 //
 
+// clang-format off
```

Comment:
Why?

Reply:
That was generated when I updated the package, when you do pub get package it just update the files it can been deleted.